css/src/components/persona.scss

```diff
@@ -0,0 +1,65 @@
+$persona-font-size-sm: $font-size-9 !default;
+$persona-font-size-md: $font-size-8 !default;
+$persona-font-size-lg: $font-size-7 !default;
+
+$persona-avatar-background-color: $alternate-background-medium !default;
+$persona-avatar-font-color: $text-invert !default;
+$persona-avatar-border-radius: $border-radius-rounded;
+
+$persona-details-font-color: $text-subtle !default;
+$persona-name-font-size: $font-size-7 !default;
+$persona-name-font-color: $text !default;
+
+$persona-gap-size: $layout-1 !default;
+
+.persona {
+	display: flex;
+	position: relative;
+	gap: $persona-gap-size;
+	font-size: $persona-font-size-md;
+
+	.persona-avatar {
+		display: flex;
+		flex-shrink: 0;
+		align-items: center;
+		justify-content: center;
+		width: 2.6666em;
+		height: 2.6666em;
+		border-radius: $persona-avatar-border-radius;
+		background-color: $persona-avatar-background-color;
+		color: $persona-avatar-font-color;
+		line-height: 1;
+
+		img,
+		svg {
+			display: block;
+			width: 100%;
+			height: auto;
+			max-height: 100%;
+			border-radius: $persona-avatar-border-radius;
+		}
+	}
+
+	.persona-details {
+		align-self: center;
```

Comment:
can `align-self: center;` be removed?

Reply:
Nope, it helps align the name nicely when there is no metadata:

![image](https://github.com/microsoft/atlas-design/assets/57374379/f3d314f5-92f8-45a5-a3c1-c11d701a4e29)

Otherwise would be this:

![image](https://github.com/microsoft/atlas-design/assets/57374379/5daebf0b-d808-46e9-a600-b4aa2a06c6a4)